core/src/main/java/fr/sncf/osrd/conflicts/SpacingResourceGenerator.kt

```diff
@@ -134,60 +100,12 @@ class SpacingRequirementAutomaton(
     private fun emitZoneRequirement(
```

Comment:
Javadoc to be updated. Technically, this method doesn't care anymore about the zone's position, it just emits a requirement for it.